src/pages/search-page/compare-terms/compare-terms.component.tsx

```diff
@@ -12,6 +12,15 @@ interface Props {
   onDeleteTerm: (id: string) => void;
 }
 
+const capitalCase = (label?: string) => {
+  if (!label) {
```

Comment:
Funker en regex for å gjøre det i en linje? 
F.eks. label.trim().replace(/^\w/, (c) => c.toUpperCase());